benchmark/bin/main.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:io';
```

Comment:
As per https://dart.dev/tutorials/server/get-started#4-run-the-app I tried `dart run` in the `benchmark` folder. Fails with

```
C:\Users\uwe\greenrobot-git\objectbox-dart\benchmark>dart run
Could not find the implicit file to run: bin\benchmark.dart.
```

Should this file be `benchmark.dart` instead?

Reply:
OK, renamed